xilinx_test/test/TestDataPack.cpp

```diff
@@ -44,64 +48,64 @@ TEST_CASE("DataPack", "[DataPack]") {
   }
 
   SECTION("Assignment copy operator") {
-    DataPack lhs(0);
+    DataPack lhs((TestType)0);
     for (int i = 0; i < kWidth; ++i) {
-      REQUIRE(lhs[i] == 0);
+      REQUIRE(lhs.Get(i) == 0);
     }
     DataPack rhs(kFillVal);
     lhs = rhs;
     for (int i = 0; i < kWidth; ++i) {
-      REQUIRE(lhs[i] == kFillVal);
+      REQUIRE(lhs.Get(i) == kFillVal);
     }
   }
 
   SECTION("Assignment move operator") {
-    DataPack lhs(0);
+    DataPack lhs((TestType)0);
     for (int i = 0; i < kWidth; ++i) {
-      REQUIRE(lhs[i] == 0);
+      REQUIRE(lhs.Get(i) == 0);
     }
     DataPack rhs(kFillVal);
     lhs = std::move(rhs);
     for (int i = 0; i < kWidth; ++i) {
-      REQUIRE(lhs[i] == kFillVal);
+      REQUIRE(lhs.Get(i) == kFillVal);
     }
   }
 
   SECTION("Index-wise assignment") {
-    DataPack lhs(0);
+    DataPack lhs((TestType)0);
     DataPack rhs(kFillVal);
     for (int i = 0; i < kWidth; ++i) {
-      REQUIRE(lhs[i] == 0);
+      REQUIRE(lhs.Get(i) == 0);
     }
     for (int i = 0; i < kWidth; ++i) {
       lhs[i] = rhs[i];
     }
     for (int i = 0; i < kWidth; ++i) {
-      REQUIRE(lhs[i] == kFillVal);
+      REQUIRE(lhs.Get(i) == kFillVal);
     }
   }
 
   SECTION("Shift operation") {
     DataPack first(kFillVal);
-    DataPack second(0);
-    first.ShiftTo<0, kWidth/2, kWidth/2>(second);
+    DataPack second((TestType)0);
```

Comment:
I'm allergic to C-style casts ;-) Use the constructor-style initialization (`TestType(0)`) or `static_cast<TestType>(0)`, either is fine with me

Reply:
You are absolutely right, I wrote this to quickly check some things and then just committed it... 
Bad habit! =D